code/_core/obj/item/weapon/ranged/spellgem/buff_armor.dm

```diff
@@ -0,0 +1,32 @@
+/obj/item/weapon/ranged/spellgem/buff_armor
+	name = "self-armoring spell gem"
```

Comment:
self-armoring doesn't seem like a good name, personally.

Reply:
renamed to self reinforcing